nightfall-deployer/contracts/Proposers.sol

```diff
@@ -29,11 +30,12 @@ contract Proposers is Stateful, Structures, Config {
 
 
   //add the proposer to the circular linked list
-  function registerProposer() external payable {
+  function registerProposer(string memory url) external payable {
```

Comment:
`State.sol` holds the list of all proposers and any associated details of each proposer as below
```  
mapping(address => LinkedAddress) public proposers;
```

Can we add url to `LinkedAddress` struct that holds proposer's info?
```
    struct LinkedAddress {
        address thisAddress;
        address previousAddress;
->    string url;
        address nextAddress;
    }
```